episodes/4-advanced-layer-types.Rmd

```diff
@@ -360,13 +366,16 @@ model.compile(optimizer='adam',
               metrics=['accuracy'])
 ```
 
+## 6. Train the model
+
 We then train the model for 10 epochs:
 
 ```python
 history = model.fit(train_images, train_labels, epochs=10,
                     validation_data=(test_images, test_labels))
 ```
 
+## 7. Perform a Prediction/Classification and 8. Measure performance
```

Comment:
We are not really performing a prediction here. Maybe we can just have a title for 7. with a statement saying this step will be skipped in this episode and then step 8 with the curves.